ingest/processors/plugin/packages.go

```diff
@@ -1,6 +1,7 @@
+// +build linux,go1.16,!go1.17,!386,!arm,!mips,!mipsle,!s390x
+
 // Code generated by scriggo command. DO NOT EDIT.
-//go:build linux && go1.16 && !go1.17
-// +build linux,go1.16,!go1.17
+//go:build linux && go1.16 && !go1.17 && linux,go1.16 && !go1.17 && !386 && !arm && !mips && !mipsle && !s390x
```

Comment:
this should probably also be at the top of the file, or removed until we decide to support go 1.17 officially